colorizer_data/writer.py

```diff
@@ -284,58 +306,73 @@ def write_data(
         Accepts numpy arrays for each file type and writes them to the configured
         output directory according to the data format.
 
+        Args:
+            tracks (`np.ndarray`): A 1D numpy array of integer track numbers, where `tracks[i]` is the track number for the `i`th object.
+            times (`np.ndarray`): A 1D numpy array of float timestamps, where `times[i]` is the time, in frames, at which the `i`th object
+                is visible.
+            centroids_x (`np.ndarray`): A 1D numpy array of float x-coordinates for object centroids.
+            centroids_y (`np.ndarray`): A 1D numpy array of float y-coordinates for object centroids.
+            outliers (`np.ndarray`): An optional 1D numpy array of boolean values, where `outliers[i]` is `True` if the `i`th object is an outlier.
+            bounds (`np.ndarray`): An optional 1D numpy array of float values. For the `i`th object, the coordinates of the upper left corner are
+                `(x: bounds[4i], y: bounds[4i + 1])` and the lower right corner are `(x: bounds[4i + 2], y: bounds[4i + 3])`.
+            write_parquet (`bool`): Whether to write the specified data as a `.parquet` file rather than the default JSON format.
```

Comment:
should be write_json, right?  binary is default now

Reply:
Good catch, thank you!